bob-common/src/configs/cluster_tests.rs

```diff
@@ -11,11 +11,13 @@ mod tests {
         let s = "
 nodes:
     - name: n1
+      tls: false
```

Comment:
I don't think, that `cluster.yaml` is a good place to turn on/off TLS support. This option should be in `node.yaml`. We don't allow scenario, when some nodes have TLS enabled and others don't